tests/test_deps.py

```diff
@@ -0,0 +1,103 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Python Test Repo Template
+# ..................................
+# Copyright (c) 2017-2024, Mr. Walls
+# ..................................
+# Licensed under MIT (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# ..........................................
+# http://www.github.com/reactive-firewall/python-repo/LICENSE.md
+# ..........................................
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__module__ = """tests"""
+
+try:
+	try:
+		import context
+	except Exception as ImportErr:  # pragma: no branch
```

Comment:
I'm on the fence about this, on one hand I agree it is unless right now. On the other hand once the improvements from issue #32 are done it might be dealt with differently.

* to strike a balance let's also add this to the #32 to keep track.